src/v3/src/types/widget.ts

```diff
@@ -189,6 +189,7 @@ export type OktaWidgetFeatures = {
   securityImage?: boolean;
   rememberMe?: boolean;
   autoPush?: boolean;
+  autoFocus?: boolean;
```

Comment:
Curious if there was already some discussion about the name of this feature. This seems in line with the other feature names but I also considered `allowAutoFocus`. Seeing the other ones I am more convinced this is the right name though.

Reply:
This is actually already an existing [feature](https://github.com/okta/okta-signin-widget/blob/master/src/models/Settings.ts#L109) that v2 uses, I assumed we will want to keep the same name so v3 is backwards compatible with existing features.

Reply:
Ah okay great, thanks for attaching the link